src/ImageSharp/Formats/Jpeg/Components/ColorConverters/JpegColorConverterVector128.cs

```diff
@@ -19,14 +18,14 @@ internal abstract partial class JpegColorConverterBase
     /// DO NOT pass test data of invalid size to these converters as they
     /// potentially won't do a bound check and return a false positive result.
     /// </remarks>
-    internal abstract class JpegColorConverterArm : JpegColorConverterBase
+    internal abstract class JpegColorConverterVector128 : JpegColorConverterBase
     {
-        protected JpegColorConverterArm(JpegColorSpace colorSpace, int precision)
+        protected JpegColorConverterVector128(JpegColorSpace colorSpace, int precision)
             : base(colorSpace, precision)
         {
         }
 
-        public static bool IsSupported => AdvSimd.IsSupported;
+        public static bool IsSupported => Vector128.IsHardwareAccelerated && Vector128<float>.IsSupported;
```

Comment:
nit: You don't need to check `Vector128<float>.IsSupported`, this is statically `true`.

The `Vector128<T>.IsSupported` is namely for generic contexts where you don't know concretely what the `T` is.

Reply:
In general you can presume that `byte`, `sbyte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `float`, and `double` are always supported.

You can presume that `nint` and `nuint` are supported on .NET 7+

We will likely add support for `Half` at some point in the future, at which point a `Vector128.AsHalf()` method will exist to "document" this guarantee of support.